content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -439,15 +456,12 @@ The SNI proxy will forward the traffic to port `443`.
       servers:
       - port:
           number: 443
-          name: tls-egress
-          protocol: TLS
+          name: https-egress
```

Comment:
Here it must be TLS, otherwise Istio will treat the traffic as HTTPS (HTTP inside TLS) and the requests will fail. Here we have HTTPS within TLS (double TLS). Let me explain it:

* Regular HTTPS with Istio Mutual TLS: **HTTP inside Istio Mutual TLS**. The gateway performs TLS termination and forwards the traffic as HTTP.
* This case: **HTTP inside TLS inside  Istio Mutulal TLS**. The gateway performs TLS termination and forwards the traffic as HTTPS, as needed.